util/libcrypto.num

```diff
@@ -4528,3 +4528,28 @@ conf_ssl_name_find                      4469	1_1_0i	EXIST::FUNCTION:
 conf_ssl_get_cmd                        4470	1_1_0i	EXIST::FUNCTION:
 conf_ssl_get                            4471	1_1_0i	EXIST::FUNCTION:
 X509_VERIFY_PARAM_get_hostflags         4472	1_1_0i	EXIST::FUNCTION:
+X509_LOOKUP_meth_get_get_by_fingerprint 4473	1_1_1	EXIST::FUNCTION:
```

Comment:
If this targets even 1.1.0, then shouldn't it be reflected here?

Reply:
Ah, quite right!  Fixing